src/Symfony/Component/DependencyInjection/Loader/XmlFileLoader.php

```diff
@@ -672,7 +672,7 @@ private function validateAlias(\DOMElement $alias, $file)
         }
 
         foreach ($alias->childNodes as $child) {
-            if (!$child instanceof \DOMElement && self::NS !== $child->namespaceURI) {
+            if (!$child instanceof \DOMElement || self::NS !== $child->namespaceURI) {
```

Comment:
Originally there was no `continue` statement (this was introduced in #29968). The condition before read like this:

```php
if ($child instanceof \DOMElement && self::NS === $child->namespaceURI) {
    throw new InvalidArgumentException(sprintf('Invalid child element "%s" defined for alias "%s" in "%s".', $child->localName, $alias->getAttribute('id'), $file));
}
```

This change ensures that we do not access a potentially not existing `$namespaceURI` property when `$child` is no `DOMElement` instance.